### ISSUE TITLE
Incorrect listing of NHTTP properties under PassThrough transport configuration 

### ISSUE THREAD

**Issue by ruwiniwj** [wso2/docs-apim#1214]
**Description:**
The following NHTTP properties have incorrectly included in the PassThrough transport configuration description of the Tuning Performance documentation [1], [2]. 

snd_t_core: 200  
snd_t_max: 250  
snd_io_threads: 16  
lst_t_core: 200  
lst_t_max: 250  
lst_io_threads: 16

These properties have been correctly mentioned in the 2.6.0 documentation [3]. Please do the needful to correct the documentation [1] and [2]

[1] https://docs.wso2.com/display/AM200/Tuning+Performance
[2] https://docs.wso2.com/display/AM210/Tuning+Performance
[3] https://docs.wso2.com/display/AM260/Tuning+Performance

**Comment by Mariangela** [comment]
@ruwiniwj , please confirm whether http.socket.timeout: should be
http.socket.timeout: 60000 OR as per 2.6.0 OR 
http.socket.timeout: 180000 as per 2.2.0, and 2.5.0

If the 2.20 value is correct is that applicable to 2.6.0 as well?

**Comment by Mariangela** [comment]
- Deleted 
```
snd_t_core: 200
snd_t_max: 250
snd_io_threads: 16
lst_t_core: 200
lst_t_max: 250
lst_io_threads: 16
```
- Updated the values to
```
worker_pool_size_core: 400
worker_pool_size_max: 500
io_buffer_size: 16384
http.socket.timeout: 180000
```
In the APIM 2.0.0 and 2.1.0 docs

**Comment by ruwiniwj** [comment]
@Mariangela We can change the 2.6.0 value to 180000.

**Comment by Mariangela** [comment]
Thanks for the clarification, @ruwiniwj . I had made a mistake in my Q. The value in 2.6.0 was 180000 and the value in 2.2.0, and 2.5.0 was 60000. So I corrected the value in 2.2.0, and 2.5.0
